app/services/bulk_upload/lettings/year2023/csv_parser.rb

```diff
@@ -97,6 +97,8 @@ def normalised_string
 
     @normalised_string = File.read(path, encoding: "bom|utf-8")
     @normalised_string.gsub!("\r\n", "\n")
+    @normalised_string.gsub!("\r\r", "\n")
+    @normalised_string.tr!("\r", "\n")
```

Comment:
Does this do the same thing as the previous line?

Reply:
Not exactly - line 100 replaces \r\r with a single newline to avoid clutter but it's probably not necessary